thirdparty/cmake_modules/koreader_thirdparty_common.cmake

```diff
@@ -31,6 +31,15 @@ if(NOT DEFINED PARALLEL_JOBS)
     math(EXPR PARALLEL_JOBS "${PROCESSOR_COUNT}+1")
 endif()
 
+# $(MAKE) is for recursive make invocations, but evidently when using another
+# generator there's no recursion. For us that other generator is ninja, but
+# maybe one day also Visual Studio or Xcode…
+if(CMAKE_GENERATOR MATCHES Makefiles)
+    set(KO_MAKE_RECURSIVE "$(MAKE)")
```

Comment:
@NiLuJe And also this one.

The system works slightly faster on first build, and definitely provide a speed-up in front on rebuilds, so I pretty much intend to switch over.

Reply:
No issues with that one ;).